src/tests/ll_verification.py

```diff
@@ -5919,6 +5942,534 @@ def ll_sec_scn_bv_01_c(transport, upperTester, lowerTester, trace):
+    #    Connection_Handle field set to the value of the CIS_Connection_Handle received in step 3.
+    status = le_accept_cis_request(transport, upperTester, cisConnectionHandle, 100)
+
+    # 5. The Upper Tester expects the IUT to send a successful Command Status.
+    success = verifyAndShowEvent(transport, upperTester, Events.BT_HCI_EVT_CMD_STATUS, trace) and (status == 0) and success
+
+    # 6. The Lower Tester receives an LL_CIS_RSP PDU from the IUT. In the message, the
+    #    CIS_Offset_Min field and the CIS_Offset_Max field are equal to or a subset of the values
+    #    received in the LL_CIS_REQ sent in step 2.
+    #
+    # NOTE: Timing is not possible to validate/control
+    s, event = verifyAndFetchMetaEvent(transport, lowerTester, MetaEvents.BT_HCI_EVT_LE_CIS_ESTABLISHED, trace, 1000)
+    success = s and (event.decode()[0] == 0x00) and success
+
+    # 7. The Lower Tester sends an LL_CIS_IND where the CIS_Offset is the time (ms) from the start of
+    #    the ACL connection event in connEvent Count to the first CIS anchor point, the CIS_Sync_Delay
+    #    is CIG_Sync_Delay minus the offset from the CIG reference point to the CIS anchor point in s,
+    #    and the connEventCount is the CIS_Offset reference point.
+    #
+    # NOTE: Timing is not possible to validate/control
+
+    # 8. The Upper Tester receives a successful HCI_LE_CIS_Established event from the IUT, after the
+    #    first CIS packet sent by the LT. The Connection_Handle parameter is the
+    #    CIS_Connection_Handle value provided in the HCI_LE_CIS_Request event.
+    #
+    # NOTE: Timing is not possible to validate/control
+    s, event = verifyAndFetchMetaEvent(transport, upperTester, MetaEvents.BT_HCI_EVT_LE_CIS_ESTABLISHED, trace)
+    success = s and (event.decode()[0] == 0x00) and success
+
+    # 9. The Lower Tester sends data packets to the IUT.
+    # 10. The Lower Tester receives an Ack.
+    # 11. Repeat steps 9 and 10 a total of 50 times.
+    txData = [0 for _ in range(10)]
+    pbFlags = 1
+    for j in range(50):
+        dataSent = writeData(transport, lowerTester, initiator.handles[1], pbFlags, txData, trace)
+        success = dataSent and success
+        if dataSent:
+            dataReceived, rxData = readData(transport, upperTester, trace)
+            success = dataReceived and (len(rxData) == len(txData)) and (rxData == txData) and success
+
+    ### TERMINATION ###
+    success = initiator.disconnect(0x13) and success
+
+    return success
+
+"""
+    LL/CIS/SLA/BV-01-C [CIS Setup Response Procedure, Slave]
+"""
+def ll_cis_sla_bv_01_c(transport, upperTester, lowerTester, trace):
+    params = SetCIGParameters(
+        SDU_Interval_M_To_S     = 7500, # 7.5 ms
+        SDU_Interval_S_To_M     = 7500, # 7.5 ms
+        ISO_Interval            = int(7.5 // 1.25), # 7.5
+        NSE                     = 2,
+        Max_PDU_M_To_S          = 60, # TODO: Supposed to be 180
```

Comment:
With BabbelCraft tracing enabled it will print the following message when using 180 bytes max pdu

> Fail to create CIS due to scheduling limitation

Reply:
Then that's something else.